dapp/src/app/actions/colony.ts

```diff
@@ -0,0 +1,54 @@
+const { providers, Wallet } = require("ethers");
+const { default: EthersAdapter } = require("@colony/colony-js-adapter-ethers");
+const { TrufflepigLoader } = require("@colony/colony-js-contract-loader-http");
+
+const { default: ColonyNetworkClient } = require("@colony/colony-js-client");
+const loader = new TrufflepigLoader();
+const provider = new providers.JsonRpcProvider("http://localhost:8545/");
+
+const getNetworkClient = async () => {
+  const { privateKey } = await loader.getAccount(0);
+  const wallet = new Wallet(privateKey, provider);
```

Comment:
For wallets there are a few different use cases we should support, as detailed here:
https://docs.ethers.io/ethers.js/html/api-wallet.html#creating-instances 

We want to make sure we can support wallets that are cold wallets, wallets from other programs like Geth, and wallets from extensions like MetaMask.

I'll create a task for this.

Reply:
https://github.com/JingyuZ/GithubColony/issues/16